parachain/runtimes/gargantua/src/xcm.rs

```diff
@@ -213,7 +212,7 @@ impl staging_xcm_executor::Config for XcmConfig {
     type MessageExporter = ();
     type UniversalAliases = Nothing;
     type CallDispatcher = RuntimeCall;
-    type SafeCallFilter = Everything;
+    type SafeCallFilter = Nothing;
```

Comment:
hmmm, so people can dispatch any xcm call to our chain? Would be good to know the associated risks. Otherwise would prefer to limit this.

Reply:
We have to add a filter for the calls we want to support, but we there's none at the moment that's why I put nothing, to block all calls

Reply:
ok so `Nothing` means block all calls? I thought it meant filter `Nothing` out. sigh, these naming schemes